typing/typemod.ml

```diff
@@ -2020,6 +2182,11 @@ let report_error ppf = function
       fprintf ppf
         "This is an alias for module %a, which is missing"
         path p
+  | Invalid_open _me ->
+      fprintf ppf "Invalid open"
+  | Cannot_eliminate_anon_module (id, sg) ->
+      fprintf ppf "The module identifier %a cannot be \
+        eliminated from %a" ident id signature sg
```

Comment:
I would add an enclosing box `@[…@]` and at least a break hint before the signature (`from@ %a`).

Reply:
The message should be improved to mention that the module identifier has been synthesized and correspond to the opened module expression.  Even better of course would be to never leak the identifier in the error message and pin-point the actual signature item whose type still refer to that identifier, but it's more difficult.